### PR TITLE
refac: Update watt-modal and Invite User dialog.

### PR DESCRIPTION
**Pull request by defectiveAi**
- Changed watt-modal title header to match UX design. This breaks padding until usages are updated.
- Small text changes.
- Increased size of Invite User dialog.
- Added overflow scrolling of user roles inside Invite User.

### CODE REVIEW

File: libs/dh/admin/feature-invite-user-modal/src/lib/dh-assignable-user-roles/dh-assignable-user-roles.component.scss [hunk 1]
```diff
@@ -14,10 +14,12 @@
...
  */
-:host {
-  overflow-y: auto;
-  display: block;
-  height: 16vw;
+.card {
+  padding: 0px;
+}
+
+.cardTitle {
```
Comment (mimse) [inline libs/dh/admin/feature-invite-user-modal/src/lib/dh-assignable-user-roles/dh-assignable-user-roles.component.scss:21]: Suggestion: Instead of adding a class to title and card component you can use the element as a selector watt-card / watt-card-title

File: libs/dh/admin/feature-invite-user-modal/src/lib/dh-assignable-user-roles/dh-assignable-user-roles.component.html [hunk 2]
```diff
@@ -35,18 +35,33 @@
...
+
+      <div class="userRolesTableContainer">
+        <watt-table
+          [dataSource]="dataSource"
+          [columns]="columns"
+          sortBy="eicFunction"
+          sortDirection="asc"
+          [selectable]="true"
+          (selectionChange)="selectionChanged($event)"
+          *transloco="let translateMarketRole; read: 'marketParticipant.marketRoles'"
```
Comment (mimse) [inline libs/dh/admin/feature-invite-user-modal/src/lib/dh-assignable-user-roles/dh-assignable-user-roles.component.html:47]: Praise: Should have been a table :100:

File: libs/dh/admin/feature-invite-user-modal/src/lib/dh-invite-user-modal.component.ts [hunk 3]
```diff
@@ -118,12 +118,12 @@ export class DhInviteUserModalComponent implements AfterViewInit, OnDestroy {
     email: [{ value: '', disabled: true }, Validators.required],
     phoneNumber: [
       '',
-      Validators.compose([
```
Comment (mimse) [inline libs/dh/admin/feature-invite-user-modal/src/lib/dh-invite-user-modal.component.ts:121]: Question: Why did you remove the compose?

**Review by mimse — CHANGES_REQUESTED** [review]

File: libs/dh/admin/feature-invite-user-modal/src/lib/dh-invite-user-modal.component.ts (hunk 3 above)
Comment (defectiveAi) [reply libs/dh/admin/feature-invite-user-modal/src/lib/dh-invite-user-modal.component.ts:121]: The required asterisk is only detected if the validation rule is directly present. Removing compose added the asterisk to phone number field.

File: libs/dh/admin/feature-invite-user-modal/src/lib/dh-assignable-user-roles/dh-assignable-user-roles.component.scss (hunk 1 above)
Comment (defectiveAi) [reply libs/dh/admin/feature-invite-user-modal/src/lib/dh-assignable-user-roles/dh-assignable-user-roles.component.scss]: Fixed.

File: libs/dh/admin/feature-edit-user-role-modal/src/lib/dh-edit-user-role-modal.component.ts [hunk 4]
```diff
@@ -58,13 +58,13 @@ import { DhPermissionsTableComponent } from '@energinet-datahub/dh/admin/ui-perm
   standalone: true,
   styles: [
     `
-      :host {
-        display: block;
+      .tabMasterData {
```
Comment (dzhavat) [inline libs/dh/admin/feature-edit-user-role-modal/src/lib/dh-edit-user-role-modal.component.ts:61]: suggestion:nitpick: consider using kebab case for CSS classes for consistency.

**Review by dzhavat — COMMENTED** [review]
praise: looks good 🚀

suggestion: the modal content in `dh-edit-user-modal.component.html` should probably also get a `...full-width` class because it's got a double margin/padding now :)

File: libs/dh/admin/feature-edit-user-role-modal/src/lib/dh-edit-user-role-modal.component.ts (hunk 4 above)
Comment (defectiveAi) [reply libs/dh/admin/feature-edit-user-role-modal/src/lib/dh-edit-user-role-modal.component.ts]: Updated naming.

**Review by dzhavat — APPROVED** [review]

**Review by mimse — APPROVED** [review]
